### ISSUE TITLE
Can't set oven

### ISSUE THREAD

**Issue by hegemmon** [oyvindwe/connectlife-ha#75]
Hi, I install your addons to HA, and connected succesfully with connectlife where I have added oven and I can set wirless, but from HA have only some sensor, no any control. Its normal?

**Comment by oyvindwe** [comment]
Yes. Unless a data dictionary with mappings are defined, all properties are only shown as sensors. You can use the `set_value` service to experiment, but I really recommend that you use the ConnectLife app to set properties, and observe what the used values are in HA.

If you need help, please submit a [JSON dump](https://github.com/oyvindwe/connectlife) for your oven!

See:
* https://github.com/oyvindwe/connectlife-ha?tab=readme-ov-file#supported-conncetlife-devices
* https://github.com/oyvindwe/connectlife-ha/tree/main/custom_components/connectlife/data_dictionaries#data-dictionaries
* https://github.com/oyvindwe/connectlife-ha?tab=readme-ov-file#service-to-set-property-values-on-sensors

**Comment by oyvindwe** [comment]
Closing due to lack of feedback
